### DISCUSSION TITLE
Tweak existing tests for new styles (5m)

### DISCUSSION THREAD

**Discussion by g105b** [SuperHyperInstantFutureTime/TrackShift#67] [Work log]
I'm going to quickly whiz through the tests that are broken due to the change in layout. No functionality should have changed, just that the test scripts are looking for an HTML layout that doesn't exist any more.

**Comment by g105b** [comment]
Well that was quick. Turns out @richardbirkin you were right, it just needed the statement field's name editing, and the message changing to "Drop your files here".

**Reply by g105b** [reply]
FYI, this fix automatically deployed your branch to https://ui-design.dev.trackshift.app/

**Comment by richardbirkin** [comment]
Great! Thanks @g105b
